### ISSUE TITLE
CMake 3.2 vs. 3.0 (debian 8)

### ISSUE THREAD

**Issue by cweiske** [v00d00/gerbera#65]
I'm trying to compile gerbera on a Debian 8 system which ships CMake 3.0.2.
gerbera says it requires 3.2. Is that correct? Could that be lowered to 3.0 to make it work on Debian?

**Comment by cweiske** [comment]
Seems that's really needed; when lowering the requirements and cmake'ing I get an error:

```
CMake Error at cmake/FindTaglib.cmake:56 (include):
  include could not find load file:

    FindLibraryWithDebug
```

**Comment by cweiske** [comment]
The cmake command builds the Makefile correctly when I lower the required CMake version to 3.0, without errors.

It still does not build on Debian 8 but that is a different topic.

**Comment by cweiske** [comment]
3.2 *is* required. With cmake 3.0 it would not build because of

```
$ make
[  1%] Building CXX object CMakeFiles/libgerbera.dir/src/action_request.cc.o
In file included from /usr/include/c++/4.9/atomic:38:0,
                 from /usr/local/src/gerbera/src/zmm/object.h:35,
                 from /usr/local/src/gerbera/src/zmm/zmm.h:35,
                 from /usr/local/src/gerbera/src/zmm/zmmf.h:35,
                 from /usr/local/src/gerbera/src/common.h:38,
                 from /usr/local/src/gerbera/src/action_request.h:38,
                 from /usr/local/src/gerbera/src/action_request.cc:32:
/usr/include/c++/4.9/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
```

Those errors are gone with CMake 3.2.

----

I could install cmake 3.6 on Debian 8 because it is in jessie-backports, see https://backports.debian.org/Instructions/

**Comment by v00d00** [comment]
We are using CMAKE_CXX_STANDARD which was added in 3.2.
